stargazer/stargazer.py

```diff
@@ -704,7 +716,7 @@ def generate_cov_precision(self, cov_name):
                     cov_text += self._float_format(md['conf_int_low_values'][cov_name]) + ' , '
                     cov_text += self._float_format(md['conf_int_high_values'][cov_name])
                 else:
-                    cov_text += self._float_format(md['cov_std_err'][cov_name])
+                    cov_text += self._float_format(md[self.sig_stat][cov_name])
```

Comment:
I understand that this approach is very clean, but given that in the future we would ideally allow to combine confidence intervals, standard errors and p-values, please add a new "else" clause for the latter.

Reply:
As in `elif self.p_values` (where, say, `self.p_values` is a flag like `self.confidence_intervals`)? I guess I don't understand what you mean, is there code for combining these three I can pattern off of?

Reply:
``elif self.sig_stat == ...`` is also OK, as you prefer.